src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/prisonapi/PrisonerApiServiceTest.kt

```diff
@@ -740,23 +689,26 @@ internal class PrisonerApiServiceTest {
       bookingId = alternativeBookingId,
       sentenceSequence = 24,
       consecutiveToSequence = 22,
+      sentenceEndDate = LocalDate.now().plusDays(10),
```

Comment:
This should have a later `sentenceEndDate` than sentence 25 in order to show up earlier in the ordered list.